src/cli/templates.hh

```diff
@@ -1092,58 +1086,19 @@ kotlin.code.style=official
 constexpr auto gAndroidMakefile = R"MAKE(
 LOCAL_PATH := $(call my-dir)
 
-## libuv.a
+## uv.a
```

Comment:
can we maintain `libuv.a` here as it is the correct library name 🙏🏼 
we often `git grep libuv` or `ag libuv` and this would be missed in that

Reply:
Good call.